SU-HSDO/suhumsci
# ARCH-315 addressed li cursor issue and added new theme screenshot

# READY FOR REVIEW

# Summary
- Addressed like-link cursor pointer for auto cursor behavior
- Added new screenshot.png file for suhumsci theme
- Addressed all npm package vulnerabilities

# Needed By (Date)
- End of sprint (8/16)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test to make sure that when you hover to the right of a link in the expanded hover menu, the cursor doesn't indicate that it links
3. Test to make sure the new screenshot.png file renders on the appearance page

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-315

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
